.github/workflows/ci.yml

```diff
@@ -228,3 +228,105 @@ jobs:
           path: /home/runner/work/avocado/avocado/bandit-output.txt
           retention-days: 1
       - run: echo "🥑 This job's status is ${{ job.status }}."
+
+
+  version_task:
+
+    name: Version task (${{ matrix.container }})
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        container: ["fedora:34",
+                    "fedora:35",
+                    "registry.access.redhat.com/ubi8/ubi",
+                    "registry.access.redhat.com/ubi9-beta/ubi",
+                    "debian:10.10",
+                    "debian:11.0",
+                    "ubuntu:21.10",
+                    "ubuntu:20.04"]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install and run avocado --version
+        run: |
+         if ! command -v apt &> /dev/null
+         then
+           python3 --version || (apt update && apt -y install python3 python3-setuptools ca-certificates)
+         fi
+         if ! command -v dnf &> /dev/null
+         then
+           python3 -c 'import setuptools' || dnf -y install python3 python3-setuptools
+         fi
+         python3 setup.py develop --user
+         python3 -m avocado --version
+
+
+  egg_task:
+
+    name: Egg task (${{ matrix.container }})
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        container: ["fedora:34",
+                    "fedora:35",
+                    "registry.access.redhat.com/ubi8/ubi",
+                    "registry.access.redhat.com/ubi9-beta/ubi",
+                    "debian:10.10",
+                    "debian:11.0",
+                    "ubuntu:21.10",
+                    "ubuntu:20.04"]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Test running avocado from eggs
+        run: |
+         if ! command -v apt &> /dev/null
+         then
+           python3 --version || (apt update && apt -y install python3 python3-setuptools)
+         fi
```

Comment:
We don't expect to have both `apt` and `dnf` so a `else if` would make more sense here.

Reply:
ok, I have changed it.